### PR TITLE
Include exclude rules & rego org experiments

### PR DESCRIPTION
**Pull request by simonbaird**
A few experimental changes. I'm trying things out.

http://file.bos.redhat.com/~sbaird/ec-policies-docs/public/ec-policies/release_policy.html is a demo of the docs.

### CODE REVIEW

File: policy/lib/include_exclude_rules.rego [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+package_excluded(package_name) {
+	any_included_in(with_wildcard(package_name), exclude_rules)
+}
+
+package_excluded(package_name) {
+	none_included_in(with_wildcard(package_name), include_rules)
+}
+
+with_wildcard(package_name) := {package_name, "*"}
```
Comment (lcarva) [inline policy/lib/include_exclude_rules.rego:22]: I see that this works as intended but I don't quite understand how. If we are always adding a wildcard to the set, wouldn't the matching always be true?

File: policy/lib/include_exclude_rules.rego [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+include_rules := result {
+	data.config
+	result := object.get(data.config, ["policy", "include_rules"], default_include_rules)
+}
+
+# Find the exclude_rules configuration or set the default if doesn't exist
+exclude_rules := result {
+	data.config
+	result := object.get(data.config, ["policy", "exclude_rules"], default_exclude_rules)
+}
```
[the remark this answers is not in the document]
Comment (simonbaird) [reply policy/lib/include_exclude_rules.rego:46]: Thanks, I'll try it.

File: policy/lib/include_exclude_rules.rego (hunk 1 above)
Comment (lcarva) [reply policy/lib/include_exclude_rules.rego:22]: Ah right. Looking at this with fresh eyes makes sense. I don't think it needs to be separated for clarity sake.

**Comment by simonbaird** [comment]
Need to rebase this on #124 now and maybe split it up. Likely to be abandoned eventually.

**Pull request closed without merging**